packages/pluginutils/src/attachScopes.ts

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line import/no-unresolved
+import * as estree from 'estree';
+
 import { walk } from 'estree-walker';
```

Comment:
This was copied from https://github.com/rollup/plugins/blob/dbececf6c72bb6276f2a80f35a49bbc7a388ace3/packages/pluginutils/types/index.d.ts#L1-L2